net/disk_cache/cache_util_unittest.cc

```diff
@@ -34,16 +35,19 @@ class CacheUtilTest : public PlatformTest {
     dir1_ = base::FilePath(cache_dir_.Append(FILE_PATH_LITERAL("dir01")));
     file3_ = base::FilePath(dir1_.Append(FILE_PATH_LITERAL("file03")));
     ASSERT_TRUE(base::CreateDirectory(cache_dir_));
-    FILE *fp = base::OpenFile(file1_, "w");
-    ASSERT_TRUE(fp != nullptr);
-    base::CloseFile(fp);
-    fp = base::OpenFile(file2_, "w");
-    ASSERT_TRUE(fp != nullptr);
-    base::CloseFile(fp);
+    int fp = -1;
```

Comment:
@yjzhang111 Would you be able to take a look at this and see if it looks like the correct way to open/close the files?

Reply:
lgtm